xml/gpu_passthru.xml

```diff
@@ -0,0 +1,503 @@
+<screen>
+&prompt.user;lspci | grep -i nvidia
+03:00.0 3D controller: NVIDIA Corporation GV100 [Tesla V100 PCIe] (rev a1)
+</screen>
+     <para>
+      With a T1000 Mobile (available on Dell 5540):
+     </para>
+<screen>
+&prompt.user;lspci | grep -i nvidia
+01:00.0 3D controller: NVIDIA Corporation TU117GLM [Quadro T1000 Mobile] (rev a1)
+</screen>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="gpu-passthru-enable-iommu">
+   <title>Enable <xref linkend="gloss-vt-acronym-iommu"/></title>
+   <para>
+    <xref linkend="gloss-vt-acronym-iommu"/> is disabled by default. You need
+    to enable it at boot time in the <filename>/etc/default/grub</filename>
+    configuration file.
+   </para>
+   <procedure>
+    <step>
+     <para>
+      For Intel-based hosts:
+     </para>
+<screen>GRUB_CMDLINE_LINUX="intel_iommu=on iommu=pt rd.driver.pre=vfio-pci"</screen>
+     <para>
+      For AMD-based hosts:
+     </para>
+<screen>GRUB_CMDLINE_LINUX="iommu=pt amd_iommu=on rd.driver.pre=vfio-pci"</screen>
+    </step>
+    <step>
+     <para>
+      When you save the modified <filename>/etc/default/grub</filename> file,
+      re-generate the main &grub; configuration file
+      <filename>/boot/grub2/grub.cfg</filename>:
+     </para>
+<screen>&prompt.sudo;grub2-mkconfig -o /boot/grub2/grub.cfg</screen>
+    </step>
+    <step>
+     <para>
+      Reboot the host and verify that <xref linkend="gloss-vt-acronym-iommu"/>
+      is enabled:
+     </para>
+<screen>&prompt.user;dmesg |  grep -e DMAR -e IOMMU</screen>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="gpu-passthru-blacklist-nouveau">
+   <title>Blacklist the Nouveau driver</title>
+   <para>
+    Because we want to assign the &nvidia; card to a VM guest, we need to avoid
+    using the card by the host opensource driver. Edit the file
+    <filename>/etc/modprobe.d/50-blacklist.conf</filename> and append the
+    following line to its end:
+   </para>
+<screen>blacklist nouveau</screen>
+  </sect2>
+
+  <sect2 xml:id="gpu-passthru-configure-vfio">
+   <title>Configure <xref linkend="gloss-vt-acronym-vfio"/> and isolate the GPU used for pass-through</title>
+   <procedure>
+    <step>
+     <para>
+      Find the card vendor and model IDs. Utilize the bus number identified in
+      <xref linkend="gpu-passthru-verify-host"/>, for example
+      <literal>03:00.0</literal>:
+     </para>
+<screen>
+&prompt.user;lspci -nn | grep 03:00.0
+03:00.0 3D controller [0302]: NVIDIA Corporation GV100 [Tesla V100 PCIe] [10de:1db4] (rev a1)
+</screen>
+    </step>
+    <step>
+     <para>
+      Create the file <filename>vfio.conf</filename> in the
+      <filename>/etc/modprobe.d/</filename> directory with the following
+      content:
+     </para>
+<screen>options vfio-pci ids=10de:1db4</screen>
+     <note>
+      <para>
+       Double check that your card does not need an extra <option>ids=</option>
+       parameter. For example, for Public Nvidia Card you need to also add the
```

Comment:
I do not understand the phrase "Public Nvidia Card", and it gets zero hits on Google. What does this mean? If we can rephrase it to something clearer, we should probably be using the nVidia entity, too.